ska_conda/pkg_defs/maude/meta.yaml

```diff
@@ -31,17 +30,19 @@ requirements:
   run:
     - python
     - setuptools
+    - requests
     - six
     - scipy
     - numpy
-    - testr
```

Comment:
`testr` is required to do `import maude; maude.test()`.

Reply:
I went back and forth on this and can obviously go back again.  It did seem to me that testr is more of a test dependence than a runtime dependence, and in a real ska runtime environment, there's always going to be a testr... but if we want to make sure that each ska package is good about also pulling in testr from the dependency tree.. that's fine too.

Reply:
If you are going to the trouble of enumerating Ska package dependencies at all, we should make them complete and not depend on being in a "real" Ska runtime.  Otherwise we could pretty much skip all the `meta.yaml` `run` requirements (except for build-time testing).

Reply:
OK, I'll move more of the test dependencies into the runtime area.